spec/controllers/api/v2/api_bookmarks_spec.rb

```diff
@@ -1,9 +1,9 @@
 require "spec_helper"
 require "controllers/api/api_helper"
 
-describe "API v2 BookmarksController", type: :request, bookmark_search: true do
-  include ApiHelper
+include ApiHelper
```

Comment:
I'd love to, but unfortunately it doesn't work inside the `describe` block. The only reason the bookmark API tests work currently is because `api_base_controller_spec.rb`, which comes before `api_bookmarks_spec.rb` alphabetically, [includes `ApiHelper` at the top level](https://github.com/otwcode/otwarchive/blob/b5bf2e7a5cd362d1e350f2be087e8049721a2484/spec/controllers/api/v2/api_base_controller_spec.rb#L4). If you try to run the spec file on its own (which I had to do for this PR), it fails.

Reply:
If you keep it inside the `describe` block and change `ApiHelper.mock_external` to `mock_external`, it should work.